<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTServerTestAppClass</class>
 <widget class="QMainWindow" name="QTServerTestAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yazilimperver Socket TCP Server Application</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QTServerTestApp.qrc">
    <normaloff>:/QTServerTestApp/data/AppIco.ico</normaloff>:/QTServerTestApp/data/AppIco.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Server Settings</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Port :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="portToListenLineEdit">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>15000</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="listenButton">
             <property name="text">
              <string>START &amp;LISTENING</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Data Configuration</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QCheckBox" name="binaryHexCheckBox">
               <property name="text">
                <string>Use Binary/Hex Data</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="addNewLineCheckBox">
               <property name="text">
                <string>Add New Line</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Message Size:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="msgSizeLineEdit">
               <property name="inputMethodHints">
                <set>Qt::ImhDigitsOnly</set>
               </property>
               <property name="text">
                <string>16</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="logoLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="QTServerTestApp.qrc">:/QTServerTestApp/data/AppLogo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Conversation With Clients</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Messages To Show :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="messageFilterComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <item>
            <property name="text">
             <string>ALL</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveHistoryButton">
           <property name="text">
            <string>SAVE HISTORY</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearHistoryButton">
           <property name="text">
            <string>CLEAR HISTORY</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="conversationList"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Client Panel</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Connected Clients :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="connectedClientsComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <item>
            <property name="text">
             <string>ALL</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="disconnectButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>DISCONNECT</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Message</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="MultiPurposeInputField" name="msgToSendLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sendButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;SEND</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MultiPurposeInputField</class>
   <extends>QLineEdit</extends>
   <header>CommonUILibrary/MultiPurposeInputField/MultiPurposeInputField.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QTServerTestApp.qrc"/>
 </resources>
 <connections/>
</ui>
